### DISCUSSION TITLE
Cleaning of messages not working

### DISCUSSION THREAD

**Discussion by niels-przybilla** [postalserver/postal#3106] [Help with using Postal]
Hello Postal Community,

we are using V 3.3.2

I changed retention from 30 days to 7 days for raw messages and meta. (it was 30 days before)
I did this 6 days ago but the old messages remain. 
Can I somehow trigger the retention rule manually to clean up?

THX and best wishes

Niels

**Comment by willpower232** [comment]
You need to update to 3.3.4 which was released to fix this problem.
